deps/build.jl

```diff
@@ -1,3 +1,8 @@
+if Sys.KERNEL === :Darwin && (!success(`command -v xcode-select`) || isempty(readchomp(`xcode-select -p`)))
```

Comment:
Should this be `is_apple()`, or have we got rid of this (I can't keep up)?

Reply:
I did it this way to avoid depwarns and avoid needing to bring in Compat during build time just for this one check. `is_apple()` is deprecated in favor of `Sys.isapple()` in 0.7 and is available as `Compat.Sys.isapple()` in Compat.

Reply:
Ah, okay. Makes sense.